README.md

```diff
@@ -38,20 +65,143 @@ juju ssh kafka/leader 'kafka.topics --bootstrap-server $BOOTSTRAP_SERVERS --list
 
 Available Kafka bin commands can be found with:
 ```
-snap info kafka --channel rock/edge
+snap info charmed-kafka --channel latest/edge
 ```
 
-### Checklist
-- [x] Super-user creation
-- [x] Inter-broker auth
-- [x] Horizontally scale brokers
-- [x] Username/Password creation for related applications
-- [x] Automatic topic creation with associated user ACLs
-- [x] Persistent storage support with [Juju Storage](https://juju.is/docs/olm/defining-and-using-persistent-storage)
-- [x] TLS/SSL encrypted connections
-- [ ]  mTLS
-- [ ] Multi-application clusters
-- [ ] Partition rebalancing during broker scaling
+### Replication
+#### Scaling up
+The charm can be scaled up using `juju add-unit` command.
+```shell
+juju add-unit kafka
+```
+
+To add a specific number of brokers, an extra argument is needed:
+```shell
+juju add-unit kafka -n <num_brokers_to_add>
+```
+
+#### Scaling down
+To scale down the charm, use `juju remove-unit` command.
+```shell
+juju remove-unit <unit_name>
+```
+
+Even when scaling multiple units at the same time, the charm uses a rolling restart sequence to make sure the cluster stays available and healthy during the operation.
+
+### Password rotation
+#### Internal operator user
+The operator user is used internally by the Charmed Kafka Operator, the `set-password` action can be used to rotate its password.
+```shell
+# to set a specific password for the operator user
+juju run-action kafka/leader set-password password=<password> --wait
+
+# to randomly generate a password for the operator user
+juju run-action kafka/leader set-password --wait
+```
+
```

Comment:
does kafka create users for relations? If so might be useful to have a section [Related application users](https://github.com/canonical/mongodb-operator/blob/1d1d90610264e9f50e72c92b37018464847b56f8/README.md?plain=1#L145)

Reply:
We don't at the moment